routes/api_v1_0.py

```diff
@@ -358,18 +358,23 @@ def get_data_v1_0():
         lat_slice = slice(0, lat_var.size, 4)
         lon_slice = slice(0, lon_var.size, 4)
 
-        time_index = ds.nc_data.timestamp_to_time_index(result['time'])
-        
-        data = ds \
-                .nc_data \
-                .get_dataset_variable(result['variable'])[time_index, result['depth'], lat_slice, lon_slice]
```

Comment:
Before, we were slicing all variables the same way...even if it was a 2D variable, we'd assume it has a depth dimension which is totally false lol.

Reply:
Thank you Nabil